example/lib/main.dart

```diff
@@ -216,9 +221,21 @@ class _UserListPageState extends State<UserListPage> {
   @override
   void initState() {
     super.initState();
+    _checkSession();
     _loadUsernames();
   }
 
+  _checkSession() async {
+    AmityUIKit().observeSessionState().listen((event) {
```

Comment:
Observe the session state to skip the login page if session state is already established